main.go

```diff
@@ -279,18 +280,24 @@ func main() {
 
 	//+kubebuilder:scaffold:builder
 
-	if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
+	if err = mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
 		setupLog.Error(err, "unable to set up health check")
 		os.Exit(1)
 	}
-	if err := mgr.AddReadyzCheck("readyz", healthz.Ping); err != nil {
+	if err = mgr.AddReadyzCheck("readyz", healthz.Ping); err != nil {
 		setupLog.Error(err, "unable to set up ready check")
 		os.Exit(1)
 	}
 
-	setupLog.Info("starting manager")
-	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
-		setupLog.Error(err, "problem running manager")
+	client, err := dynamic.NewForConfig(ctrl.GetConfigOrDie())
```

Comment:
Don't think it matters, but not sure do we want to get the `rest.Config` from the `mgr` instead here since it's available through it
```suggestion
	client, err := dynamic.NewForConfig(mgr.GetConfig())
```

Reply:
Sure no problem